test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -83,7 +90,7 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_peer_failure, dc_mlx5, "dc_mlx5")
 
 
 test_ucp_peer_failure::test_ucp_peer_failure() :
-    m_am_rx_count(0), m_err_count(0), m_err_status(UCS_OK)
+    m_am_rndv_descs(), m_am_rx_count(0), m_err_count(0), m_err_status(UCS_OK)
```

Comment:
do we really need to call the constructor of m_am_rndv_descs explicitly?

Reply:
No, we don't need it. Removed.